pytest_reportportal/service.py

```diff
@@ -92,10 +91,10 @@ def init_service(self, endpoint, project, uuid, log_batch_size,
                 project=project,
                 token=uuid,
                 error_handler=self.async_error_handler,
-                log_batch_size=log_batch_size,
-                verify_ssl=verify_ssl
+                log_batch_size=log_batch_size  # ,
+                # verify_ssl=verify_ssl
             )
-            self.project_settiings = self.RP.rp_client.get_project_settings() if self.RP else None
+            self.project_settiings = None  # self.RP.rp_client.get_project_settings() if self.RP else None
```

Comment:
I had to implement the fix of PR https://github.com/reportportal/agent-python-pytest/issues/112 in order to test my changes in a live env - otherwise, I would get an exception right away